docs/guides/main.rst

```diff
@@ -139,6 +146,11 @@ place to start::
     directory: ~/music
     library: ~/data/musiclibrary.db
 
+For Windows user, this would look like::
+
+    directory: "D:\\My Music Files"
+    library: "D:\\My Music Files\\Musiclibrary.db"
```

Comment:
Also, for what it's worth, you can still use the unixy-looking paths shown just above: `~/music` will work just fine and use a directory called "music" in your `%HOME%` directory.